.github/workflows/gource.yml

```
@@ -0,0 +1,31 @@
+name: Gource
+
+on:
+  push:
+    branches:
+      - gource
+  schedule:
+    - cron:  '15 3 1 * *'
+  release:
+    types:
+      - created
+
+jobs:
+  action:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout'
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: 'Gource Action'
+        uses: nbprojekt/gource-action@v1.0.1
```

`v1` Tag can be used now
```suggestion
        uses: nbprojekt/gource-action@v1
```